admin/api_projects.go

```diff
@@ -362,16 +362,16 @@ type ProjectsApi interface {
 	removeProjectUserExecute(r RemoveProjectUserApiRequest) (*http.Response, error)
 
 	/*
-			SetProjectLimit Set One Project Limit
```

Comment:
Diff is because goimports and gofmt aren't compatible. 
If we do not set single tool we going to get CI/Failures. 
This file changes come from gofmt